editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1381,7 +1384,8 @@ Error EditorSceneImporterGLTF::_parse_images(GLTFState &state, const String &p_b
 			}
 		}
 
-		ERR_FAIL_COND_V_MSG(img.is_null(), ERR_FILE_CORRUPT, "glTF: Couldn't load image with its given mimetype: " + mimetype);
+		ERR_FAIL_COND_V_MSG(img.is_null(), ERR_FILE_CORRUPT,
+				vformat("glTF: Couldn't load image index '%d' with its given mimetype: %s.", i, mimetype));
 
```

Comment:
Never mind, works like a charm :)

Reply:
Yeah this would result in a failure in a different case where e.g. the mime type is correct but the data is broken, and thus the image parsing would fail.

I considered that at this point it's better to abort with a big red error, and we can evaluate such cases if/when they arise.

With the above we're making an exception for `image/gif` but other kinds of invalid glTF files should still be treated as invalid.